analyzeme/src/profiling_data.rs

```diff
@@ -261,6 +169,11 @@ impl ProfilingDataBuilder {
         )
         .unwrap();
 
+        string_table.alloc_metadata(&*format!(
```

Comment:
What's this for?

Reply:
The string table contains a "metadata" entry that stores some global information. The `start_time` in there is used for mapping the timings in each event to a `SystemTime`, so we need it for test cases that look at those times.